client/src/Component/CommonComponent/Input/style.css

```
@@ -0,0 +1,23 @@
+form {
+    display: flex;
+    flex-direction: column;
+}
+label{
+    align-self: flex-start;
+    font: 1.4em Roboto;
+    padding: 0.3em;
+  
+}
+input{
+    color: #646769;
+background: #EEEEEE;
+border-radius: 3px;
+display: inline-block;
+border-style: none;
+border: rgba(144,144,144, 0.6) 0.8px;
+padding: 0.3em;
+margin: 0.3em;
+font: 1.4em Lato;
```

It looks great. I have one suggestion. The font size of the placeholder is a little bit big for a placeholder and the padding is quite small. Try to change it to:
```suggestion
font: 1.2em Lato;
padding: 0.8em;
```
And tell me what you think!

---

hadn't seen the pull request description :smile: